vimrc

```diff
@@ -158,3 +158,6 @@ set diffopt+=vertical
 if filereadable($HOME . "/.vimrc.local")
   source ~/.vimrc.local
 endif
+
+set secure
```

Comment:
What do you think of setting these settings only when `.git/safe` is in place in the repository?

I imagine it were to looks something like:

```
if isdirectory(getcwd() . ".git/safe")
  set secure
  set exrc
endif
```

That way - we can be sure that the users vim configuration isn't going to be compromised.

Thoughts?